src/openrct2/screens/game/GameScreen.cpp

```diff
@@ -0,0 +1,50 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2019 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#include "GameScreen.h"
+
+#include "../../Context.h"
+#include "../../Game.h"
+#include "../../GameState.h"
+#include "../../Input.h"
+#include "../../OpenRCT2.h"
+#include "../../Version.h"
+#include "../../audio/audio.h"
+#include "../../config/Config.h"
+#include "../../core/Console.hpp"
+#include "../../drawing/Drawing.h"
+#include "../../interface/Screenshot.h"
+#include "../../interface/Viewport.h"
+#include "../../interface/Window.h"
+#include "../../localisation/Localisation.h"
+#include "../../network/network.h"
+#include "../../scenario/Scenario.h"
+#include "../../scenario/ScenarioRepository.h"
+#include "../../ui/UiContext.h"
```

Comment:
It was a pretty quick draft, and no, not at this point. I want to eventually deduplicate some code that also the title screen uses because both are more or less just simulating the park with a few minor differences.

Reply:
I agree, this does seem a bit odd if this is a new file.